ccpp/data/GFS_typedefs.meta

```diff
@@ -2767,6 +2933,18 @@
   units = flag
   dimensions = ()
   type = logical
+[rrfs_smoke]
+  standard_name = flag_for_rrfs_smoke_coupling
```

Comment:
`do_smoke_coupling`

Reply:
I've renamed this variable in the code, and I'm testing the code now.